### DISCUSSION TITLE
Small Wiki edit -- Amend for new "System Settings" (formerly System Preferences")

### DISCUSSION THREAD

**Discussion by coquizen** [asmvik/yabai#2094] [General]
Since Sonoma? Apple changed and shifted around the formerly known System Preferences. This[section](https://github.com/koekeishiya/yabai/wiki/Configuration#status-bar) should have the addendum of the new location for hiding the menu bar. ( `System Settings` -> `Control Center` -> `Automatically Hide and Show the menu bar` )

**Comment by asmvik** [comment]
Updated with instructions for various macOS versions.
